### DISCUSSION TITLE
come some an error when start Mariadb.

### DISCUSSION THREAD

**Discussion by Dilihapu** [termux/termux-packages#17387] [General]
Mysql is unable to log even though it appears to be working correctly after first starting. When I check again, it shows an error like in the screenshot.
How fix it!
![Screenshot_20230708-075743](https://github.com/termux/termux-packages/assets/125900049/6fcef5db-acf0-4b67-9a63-385a445a977f)

**Comment by sylirre** [comment]
Are you trying to run 2 mysql instances in parallel? Seems like this case.
